R/sp.R

```diff
@@ -170,32 +173,64 @@ setAs("sf", "Spatial", function(from) {
 		data.frame(from), match.ID = FALSE)
 })
 
+#' @name as
 #' @rdname coerce-methods
 #' @aliases coerce,sfc,Spatial-method
 setAs("sfc", "Spatial", function(from) as_Spatial(from))
 
-# setAs("sfg", "Spatial", function(from) as(st_sfc(from), "Spatial"))
-##  doesn't work for:
-## as(st_point(0:1), "Spatial")
+# create empy class
+setClass("XY")
+setAs("XY", "Spatial", function(from) as(st_sfc(from), "Spatial"))
```

Comment:
why not use `setAs("sfg", "Spatial",...etc)`  here? This would then also work for `XYM` points.